### ISSUE TITLE
Phishing Warning On Emails Sent From Web Express

### ISSUE THREAD

**Issue by alexfinnarn** [CuBoulder/express#1524]
## Context

Currently, emails received in Gmail have a phishing warning on them.

This is probably a minor thing, but people with Gmail accounts will see a warning message at the top of their email about phishing claiming that the gmail account cannot be verified.

Since a lot of people use Gmail, I think it would make newsletter sign-ups look better if they didn't have that warning. I'm not even sure the email accounts even have to be at Gmail or why they couldn't send the email through the web server using the SMTP module or something similar.

https://www.drupal.org/project/smtp

**Comment by alexfinnarn** [comment]
I know there was a way to log emails on express.local to a "/var/log/mail" directory or something so testing should use that method to check that emails were sent. 

Trying to look at an email in Gmail would require a test Google account.
